### DISCUSSION TITLE
post/use-secrets-in-unit-tests

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#57] [Announcements]
# post/use-secrets-in-unit-tests

Mark Heath's Development Blog

https://markheath.net/post/use-secrets-in-unit-tests

**Comment by VladislavAntonyuk** [comment]
Hi!
How it can be processed later on CI?

**Reply by markheath** [reply]
Ah yes, probably should have mentioned that in the post! I'd just go with environment variables for CI, (just chain on AddEnvironmentVariables to the configuration builder).
